public/locales/en/translation.json

```diff
@@ -10,7 +10,14 @@
     "Previous page": "Previous page",
     "Coming soon": "Coming soon",
     "Cancel": "Cancel",
-    "Save": "Save"
+    "Save":"Save",
+    "Warning":"Warning",
+    "Calling":"Calling",
+    "Operators":"Operators",
```

Comment:
We already have these strings: `"Operators.Operators"`, `"Phonebook.Phonebook"`, ...